src/client/components/sideMenubar/sideMenu.styles.css

```diff
@@ -1,21 +1,25 @@
 @import url('https://fonts.google.com/specimen/Chakra+Petch?preview.text=GuideIT&preview.text_type=custom');
 
+.side_menu_div {
+  width: 100px;
+}
+
 li {
   background-color: #298f8a;
-  width: 60px;
   text-align: center;
   padding-top: 3px;
   border-right: solid 5px transparent;
+  line-height: 0.8;
 }
 
 .side_menu_whiteborder:hover {
   border-right-color: white;
 }
 
 .side_menu_ul {
+  width: 100%;
   list-style: none;
-  padding-left: 15px;
-  padding-right: 15px;
+  margin: 10px auto;
 }
 
```

Comment:
I think, a little more padding could be added to avoid the texts touching border :)

Reply:
Thanks Tithi for reviewing and giving suggestions :)

Reply:
hi @TITHI1244 and @cph-kiwi I tried to make changes as per your comment ... Thamks :)